Helper/Data.php

```diff
@@ -1341,15 +694,6 @@ public function getPspReferenceSearchUrl($pspReference, $liveEnvironment): strin
         );
     }
 
-    /**
-     * @param $shopperReference
-     * @return string
-     */
-    public function padShopperReference(string $shopperReference): string
```

Comment:
I don't think `padShopperReference()` method belongs to `PlatformInfo` class. It pads the customer ID to make it at least 3 digits as Checkout API requires this format. Could you please reconsider refactoring of this method?